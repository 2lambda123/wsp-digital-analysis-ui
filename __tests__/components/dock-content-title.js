/* global describe, it, expect */

import React from 'react'
import renderer from 'react-test-renderer'

import Title from '../../lib/components/dock-content-title'

describe('Component > Title', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Title>Title text</Title>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
