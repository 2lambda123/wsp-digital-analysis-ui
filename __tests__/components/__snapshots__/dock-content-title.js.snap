exports[`Component > Title renders correctly 1`] = `
<pure(Component)>
  <h3
    className="DockContent-Title lead">
    Title text
  </h3>
</pure(Component)>
`;
